conf/hadoop2/spark-core-template.xml

```diff
@@ -0,0 +1,62 @@
+<configuration>
```

Comment:
For core-site.xml, we can create a link from the hadoop version of the file into the spark/conf directory. I'm wondering if we can do something similar with hbase-site.xml.
This is where core-site.xml is done: https://github.com/GoogleCloudPlatform/bdutil/blob/master/extensions/spark/install_spark.sh#L63